lnwallet/channel.go

```diff
@@ -1804,6 +1804,24 @@ func (lc *LightningChannel) restoreStateLogs(
 			return err
 		}
 
+		// Earlier versions did not write the log index to disk for fee
+		// updates, so they will be unset. To account for this we set
+		// them to to current update log index.
+		if payDesc.EntryType == FeeUpdate && payDesc.LogIndex == 0 {
+			payDesc.LogIndex = lc.localUpdateLog.logIndex
+			walletLog.Debugf("Found FeeUpdate on "+
```

Comment:
do we also want to add `lc.localUpdateLog.logIndex > 0` to the condition above? otherwise it seems this log will be triggered if a fee update is the first entry in the log. There'd be no change in behavior, as the final log index will again be set to zero, though i wonder if the log message might confuse people

Reply:
Agreed, fixed!